src/hooks/usePublish.ts

```diff
@@ -115,6 +115,7 @@ function usePublish(): UsePublish {
           const origComputePrivacy: ServiceComputePrivacy = {
             allowRawAlgorithm: true,
             allowNetworkAccess: false,
+            allowAllPublishedAlgorithms: true,
```

Comment:
wait, upon publish we actually want this to be `false`

Reply:
oh ok, my bad, thanks for fixing